SD-K9/src/app/components/outdoor-map/outdoor-map.component.ts

```diff
@@ -89,17 +95,20 @@ export class OutdoorMapComponent implements OnInit, OnDestroy, Map {
         center: this.currentPos,
         zoom: 15,
         mapTypeId: google.maps.MapTypeId.ROADMAP,
-        disableDefaultUI: true
+        disableDefaultUI: true,
+        clickableIcons: false //turns off clickable POI
```

Comment:
Why did you turn off Google Maps inforwindows for POI

Reply:
I did that because if you click on a POI, it would simply show the address which is redundant information. Instead, I created a new InfoWindow that allows you to get that address and set it as a Starting Point or Destination Point.